client/src/app/tabs/getEditMenu.js

```diff
@@ -12,6 +12,33 @@ import Flags, { DISABLE_ADJUST_ORIGIN } from '../../util/Flags';
 
 const SPACE_KEY = ' ';
 
+const COLORS = [
+  {
+    title: 'Default',
+    fill: 'white',

```

Comment:
Proposal: Set `fill` and `stroke` to undefined instead to reset colors (to default, cf. https://github.com/bpmn-io/bpmn-js-color-picker/blob/master/colors/ColorPopupProvider.js#L20.

Reply:
Updated. Should be good to go now